src/stores/languageStore.ts

```
@@ -41,27 +42,27 @@ export const useLanguageStore = create<LanguageState>((set, get) => ({
         // This case handles if the 'languages' key is missing or functionResponse itself is null/undefined
-        console.error('No languages data found in function response:', functionResponse);
-        throw new Error('No languages data returned from the server.');
+        console.error(
+          "No languages data found in function response:",
+          functionResponse
+        );
+        throw new Error("No languages data returned from the server.");
       }
 
       set({ languages, loading: false, hasFetched: true, error: null });
     } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)